docs/src/quickstart/deploy_contract.md

```diff
@@ -1,33 +1,41 @@
 
+In this guide, you will use [Foundry](https://book.getfoundry.sh/) which is a very lightweight set of command-line tools for managing smart contract development. If you already have Foundry installed, head straight to the next section. If you need to install Foundry, head to [getfoundry.sh](https://getfoundry.sh) and follow the installation steps.
 
-### Setting up a Sepolia user account
+> Note: If you are a seasoned smart contract developer, feel free to follow the tutorial using your own toolchain.
 
-To deploy our contract to the Sepolia testnet we will need to have an Ethereum account that has some SepoliaETH. SepoliaETH has no real value but it is still a resource that is needed as a basic line of defense against spamming the testnet. If you don't have any SepoliaETH yet, you can [request some SepoliaETH](https://ethereum.org/en/developers/docs/networks/#sepolia) from one of the faucets that are listed on the ethereum.org website.
+## Deploying to a local network
 
+Foundry has its own local network called [Anvil](https://book.getfoundry.sh/reference/anvil/). You can use it to create a local blockchain on your computer. Open a terminal and run the following very simple command:
 
-> **IMPORTANT**: It is good practice to **never** use an Ethereum account for a testnet that is also used for the actual Ethereum mainnet.
+```sh
+anvil 
+```
+
+You will see some ASCII art and configuration details in the terminal. Anvil creates a set of accounts that you can use on this network. The account addresses and private keys are displayed in the console (**never** use these accounts to interact with any live network). You will also see a line reading `listening on 127.0.0.1:8545`. This indicates that your local node is listening for HTTP traffic on your local network on port 8545 - this is important because this is how you will send the necessary information to your node so that it can be added to the blockchain, and how you will interact with the contract after it is deployed.
 
+> Note: Anvil needs to keep running throughout this tutorial - if you close the terminal your blockchain will cease to exist. Once Anvil has started, open a **new terminal** tab/window to run the rest of the commands in this guide.
 
 ### Making the deployment transaction
 
-Let's recall that we finished our guest book in the previous chapter with the following code.
+In the previous guide you wrote the following contract, and compiled it using `./fe build guest_book.fe --overwrite` to obtain the contract bytecode. This compilation stage converts the human-readable Fe code into a format that can be efficiently executed by Ethereum's embedded computer, known as the Ethereum Virtual Machine (EVM). The bytecode is stored at an address on the blockchain. The contract functions are invoked by sending instructions in a transaction to that address.
 
-```fe
+```rust
```

Comment:
Note that snippets of `fe` are automatically checked to compile during CI. With Fe being a fast evolving language that slightly breaks compatibility from version to version, this ensures that the examples in our docs keep functioning.

Reply:
ok, thanks